templates/test-sw.hbs

```diff
@@ -44,22 +44,22 @@
 
     <script>
       mocha.setup({
-        ui: 'bdd'
+        ui: 'bdd',
+        timeout: 10000,
       });
 
       chai.should();
-      const expect = chai.expect;
+      window.expect = chai.expect;
     </script>
 
     <!-- Package-specific tests matching packages/<pkg>/test/browser/*.js -->
 
     <script>
     describe('{{pkg}} > test/sw/*.js', function() {
-      {{#each scripts}}
-      it(`should run '{{this}}' sw tests`, function() {
```

Comment:
This looks like you've removed the ability to test multiple test files in favor of a single file. Could we support both options? I.e. some files can be plain JS and some files can be bundled JS builds and the testing knows to look in two locations and load them accordingly?